pkgs/applications/radio/jtdx/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitHub, asciidoc, asciidoctor, autoconf, automake, cmake,
+  docbook_xsl, fftw, fftwFloat, gfortran, libtool, libusb1, qtbase,
+  qtmultimedia, qtserialport, qttools, boost, texinfo, wrapQtAppsHook, jtdxhamlib, libsForQt5 }:
+
+stdenv.mkDerivation rec {
+  pname = "jtdx";
+  version = "2.2.159";
```

Comment:
Is this a tag? Can it be used as the rev in `fetchFromGitHub` (e.g. `rev = "v${version}";`)?